spec/vcr_cassettes/manageiq/providers/hawkular/middleware_manager/middleware_datasource.yml

```diff
@@ -12,7 +12,7 @@ http_interactions:
       Accept-Encoding:
       - gzip, deflate
       User-Agent:
-      - rest-client/2.0.0.rc1 (darwin15.3.0 x86_64) ruby/2.2.4p230
+      - rest-client/2.0.0.rc1 (linux-gnu x86_64) ruby/2.3.0p0
```

Comment:
Why is it needed to rerecord the cassette? from the diff it looks like max wait time was fetched in the previous cassette version as well.